native/cocos/audio/common/utils/primitives.cpp

```diff
@@ -317,7 +317,7 @@ size_t nonZeroStereo16(const int16_t *frames, size_t count) {
  */
 #define COPY_FRAME_BY_MASK(dst, dmask, src, smask, count, zero) \
     {                                                           \
-        uint32_t bit, ormask;                                   \
+        int32_t bit, ormask;                                    \
```

Comment:
Why need to change to int32_t?

Reply:
A warning is generated when taking the inverse of an unsigned int variable, but in this case, both variables are used for bitwise operations and do not impact the result.